### DISCUSSION TITLE
How can I build this project as a web component and not a JS app

### DISCUSSION THREAD

**Discussion by saado-n** [IBM/pwa-lit-template#493] [Q&A]
Hi,

The default `rollup.config.js` builds this project as a app.
Is there a way to build it as a WC so at the end of the build the output will be 1 JS file.

Then I my host app, I will use it like this:
```
<app-index id="appIndex"></app-index>
<script type="module" src="./elements/core-dashboard-lit-web-component.js">
```

Thank you

**Comment by abdonrd** [comment]
Sorry, but this template has no way to package the application as a single file.

For that you should change the Rollup configuration, but I don't know how to get what you are looking for.
The Rollup documentation should help you with that: https://rollupjs.org/guide/en/
